integration-tests/client/chainlink.go

```diff
@@ -743,6 +743,56 @@ func (c *Chainlink) ImportVRFKey(vrfExportKey *VRFExportKey) (*VRFKey, *http.Res
 	return vrfKey, resp.RawResponse, err
 }
 
+// MustCreateDkgSignKey creates a DKG Sign key on the Chainlink node
+// and returns error if the request is unsuccessful
+func (c *Chainlink) MustCreateDkgSignKey() (*DKGSignKey, error) {
```

Comment:
Naming: `MustCreateDKGSignKey`.

Also same note re: `Must` prefix applies.

Reply:
answered, waiting for reply from SET team

Reply:
Created refactoring ticket to address this - https://smartcontract-it.atlassian.net/browse/QA-1106